Src/Generic/Viewables/CheckmarkLabel.swift

```diff
@@ -0,0 +1,55 @@
+//
+//  CheckmarkLabel.swift
+//  ugglan
+//
+//  Created by Axel Backlund on 2019-04-10.
+//
+
+import Flow
+import Form
+import Foundation
+import UIKit
+
+struct CheckmarkLabel {
+    let styledTextSignal: ReadWriteSignal<StyledText>
+    
+    init(styledText: StyledText) {
+        self.styledTextSignal = ReadWriteSignal(styledText)
+    }
+}
+
+extension CheckmarkLabel: Viewable {
```

Comment:
Does this have to be so specific, couldn't it be just a label with prepending icon?